autoscoper/src/ui/NewTrialDialog.cpp

```diff
@@ -74,6 +81,11 @@ NewTrialDialog::NewTrialDialog(QWidget* parent)
   for (int i = 0; i < nbVolumes; i++) {
     VolumeBox* box = new VolumeBox();
     box->widget->groupBox_Volume->setTitle("Volume " + QString::number(i + 1));
+
+#ifdef Autoscoper_COLLISION_DETECTION
+    addMeshFileFields(box);
```

Comment:
You could add the fields in the UI file and instead hide (or disable) them here.